qemu/tests/qemu_guest_agent.py

```diff
@@ -4383,7 +4383,7 @@ def get_qga_pkg_path(self, qemu_ga_pkg, test, session, params, vm):
             vm.copy_files_to(gagent_host_path, self.gagent_guest_dir)
             qemu_ga_pkg_path = r"%s\%s" % (self.gagent_guest_dir, qemu_ga_pkg)
         elif self.gagent_src_type == "virtio-win":
-            vol_virtio_key = "VolumeName like '%virtio-win%'"
```

Comment:
@leidwang is this a must for wmic replacement? It'll lead to failure.
After applying the two patches, lead to one failure.

C:\>wmic logicaldisk where (VolumeName='%virtio-win*')
No Instance(s) Available.

C:\>wmic logicaldisk where (VolumeName like '%virtio-win%') get DeviceID
DeviceID  
E: 

I don't know why still consuming the 'wmic' cmd in my tests.